obspy/io/mseed/core.py

```diff
@@ -785,6 +785,16 @@ def _write_mseed(stream, filename, encoding=None, reclen=None, byteorder=None,
                 trace_attr['encoding'] = 1
             elif trace.data.dtype.type == np.dtype(native_str('|S1')).type:
                 trace_attr['encoding'] = 0
+            #int64 data not supported; if possible, downcast to int32, otherwise create error message
+            elif trace.data.dtype.type == np.int64:
+                #check if data can be safely downcast to int32
```

Comment:
PEP8: one space after comment sign (you should consider using something like http://flake8.pycqa.org/en/latest/ or https://github.com/ambv/black to check if your changed code conforms to PEP8)